consumer-backend/consumer.js

```diff
@@ -23,30 +21,41 @@ try {
   // service bindings. If so use that connect info
   kafkaConnectionBindings = serviceBindings.getBinding('KAFKA', 'kafkajs');
 } catch (err) {
-  // No service bindings. TODO: better error handling here
   kafkaConnectionBindings = {
     brokers: [process.env.KAFKA_HOST || 'my-cluster-kafka-bootstrap:9092']
   };
+  kafkaConnectionBindings.ssl = true;
```

Comment:
I think this line might still need to be inside the `  if (process.env.KAFKA_SASL_MECHANISM === 'plain') {` and `} else if (process.env.KAFKA_SASL_MECHANISM === 'oauthbearer') {` sections because in the case where the KAKFA_SASL_MECHANISM is not set, in the past `kafkaConnectionBindings.ssl = true` would not have been set.

I don't know if this would affet the non-RHOAS case (ie when Kafka is isntalled with the operator) but it could.

Reply:
I did it mostly for code re-usablitiy which not a good Idea on configuration like this and I cant think a scenario this would break, so better safe than sorry. Reverting it.